HW_16/HW_16.py

```diff
@@ -0,0 +1,85 @@
+website = "https://djinni.co/"
+
+# Ancestor
+xpath_selector_1 = "//a[@data-analytics-param='login_homepage']/ancestor::li"
+
+# Ancestor-or-self
+xpath_selector_2 = "//a[@class='navbar-brand']/ancestor-or-self::a"
+
+# Attribute
+xpath_selector_3 = "//attribute::target"
+css_selector_1 = "[target]"
+
+xpath_selector_4 = "//a/attribute::rel"
+
+# Child
+xpath_selector_5 = "/html/body/div[1]/div[1]/div[1]/h1"
+xpath_selector_6 = "//child::h1"
+
+xpath_selector_7 = "//div[@class='container container_landing']/child::p"
+css_selector_2 = "p:only-child"
+
+# Descendant
+xpath_selector_8 = "//div[@align='center']/descendant::a"
+
+# Descendant-of-self
+xpath_selector_9 = "//div[@class='jobs-push-briefly']/descendant-or-self::p"

```

Comment:
![image](https://user-images.githubusercontent.com/25909810/226979626-3ef0774c-3cf3-435a-8c92-6baa52db62bf.png)


I just open "https://djinni.co/" and cant find lots of locators. Do I nee some preconditions to make it works?

Reply:
![image](https://user-images.githubusercontent.com/126335301/226987585-2c4f5648-169d-45a2-adab-651637c2335e.png)